src/app/package.json

```diff
@@ -44,6 +44,7 @@
     "jest-watch-typeahead": "^0.2.1",
     "leaflet": "~1.3.1",
     "mini-css-extract-plugin": "0.5.0",
+    "nuka-carousel": "^4.5.4",
```

Comment:
I led by poor example; let's pin our package versions to at least minor versions `~`

Reply:
Ok. FYI this was just the default given by `yarn add nuka-carousel`, but I switched it regardless.